miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/MiniAppWebView.kt

```diff
@@ -45,6 +47,10 @@ internal class MiniAppWebView(
             ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.MATCH_PARENT
         )
 
+        whenAdMobProvided {
+            MobileAds.initialize(context)
```

Comment:
Maybe we should require that the Host App call `MobileAds.initialize` in their own App initialization code? I was thinking that makes more sense because they also have to add the ad-mob dependency and the Ad Mob Application ID. It also gives them more control over when it's initialized. What do you think?

Are there any problems if `MobileAds.initialize` is called multiple times?

Reply:
There is no problem on calling `MobileAds.initialize` multiple times but let the hostapp control is good for reducing workload on MiniAppWebView -> then we need to handle the exception of AdMob if it has not been initialized.